extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -104,8 +105,9 @@
     HibernateOrmConfig hibernateConfig;
 
     @BuildStep
-    HotDeploymentWatchedFileBuildItem configFile() {
-        return new HotDeploymentWatchedFileBuildItem("META-INF/persistence.xml");
+    List<HotDeploymentWatchedFileBuildItem> hotDeploymentFiles() {
+        return Arrays.asList(new HotDeploymentWatchedFileBuildItem("META-INF/persistence.xml"),
+                new HotDeploymentWatchedFileBuildItem("import.sql"));
```

Comment:
`import.sql` should be replaced by the value read from the Hibernate Configuration. This piece of code was previously here - https://github.com/quarkusio/quarkus/blob/beac5f75ab0b67979e0de65b5107dd120ab0b3fb/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java#L440-L450

Reply:
Good point, thanks @machi1990!